users/create.go

```diff
@@ -26,6 +26,9 @@ import (
 	"github.com/trackit/trackit-server/db"
 	"github.com/trackit/trackit-server/models"
 	"github.com/trackit/trackit-server/routes"
+	"github.com/aws/aws-sdk-go/service/marketplacemetering"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Please sort your imports:

1. Go imports
2. _Blank line_
3. External imports (all imports not ending with `trackit-server/*`)
4. _Blank line_
5. Our imports (all imports ending with `trackit-server/*`)